content/en/blog/_posts/2025-xx-xx-kubernetes-1-35-csi-sa-tokens-secrets-field-beta.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: blog
+title: "Kubernetes v1.35: A Better Way to Pass Service Account Tokens to CSI Drivers"
+date: 2025-xx-xxT10:30:00-08:00
+slug: kubernetes-v1-35-csi-sa-tokens-secrets-field-beta
```

Comment:
We probably need `draft: true` here??


```suggestion
slug: kubernetes-v1-35-csi-sa-tokens-secrets-field-beta
draft: true
```

ref: https://kubernetes.io/docs/contribute/blog/article-submission/#article-scheduling

Reply:
@everpeace marked it as draft!